src/backends/pipewire/device.rs

```diff
@@ -59,7 +59,7 @@ impl AudioInputDevice for PipewireDevice {
             samplerate: 48000.0,
             channels: 0b11,
             exclusive: false,
-            buffer_size_range: (None, None),
+            buffer_size_range: self.buffer_size_range()?,
```

Comment:
I don't think this is okay, as this means any call to `default` is going to call the CoreAudio API. `(None, None)` already means "use the whole range of available sizes".